src/modules/addReserve.js

```diff
@@ -0,0 +1,18 @@
+const addReserve = (dataid, name, sDate, eDate) => {
+  fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/cXyK0mK0Ha1BkQqOi8Kz/reservations',
+    {
+      method: 'post',
+      body: JSON.stringify({
+
+        item_id: dataid,
+        username: name,
+        date_start: sDate,
+        date_end: eDate,
+
+      }),
+      headers: { 'content-type': 'application/json' },
+    })
+    .then((resc) => resc.text());
```

Comment:
- [ ] As stated 😃 by the previous reviewer kindly 😊 try to implement async and await with the fetch method to send and receive data from the API. The async and await keywords enable asynchronous, promise-based behavior to be written in a cleaner style.💯